### PR TITLE
feat(action-menu, combobox, dropdown, input-date-picker, popover, tooltip): Add 'overlayPositioning' property. #2069

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #2069 #1913

## Summary

feat(action-menu, combobox, dropdown, input-date-picker, popover, tooltip): Add 'overlayPositioning' property. #2069

Describes the type of positioning to use for the overlaid content. If your element is in a fixed container, use the 'fixed' value.

### CODE REVIEW

**Review by jcfranco — COMMENTED** [review]
Is there a more user-friendly name/values we could use instead of exposing the option directly from Popper? `positioning`, `positionStrategy`, `contentPlacement`, `<something-else>`?

**Comment by driskull** [comment]
I'm ok with `positionStrategy`. Is that ok?

File: src/components/calcite-action-menu/calcite-action-menu.tsx [hunk 1]
```diff
@@ -87,6 +87,9 @@ export class CalciteActionMenu {
     this.calciteActionMenuOpenChange.emit(open);
   }
 
+  /** Describes the positioning strategy to use. If your reference element is in a fixed container, use the fixed strategy. */
```
Comment (jcfranco) [inline src/components/calcite-action-menu/calcite-action-menu.tsx:90]: Nitpick: will users understand what 'strategy' means here?

**Review by jcfranco — APPROVED** [review]
👍🏼

File: src/components/calcite-action-menu/calcite-action-menu.tsx (hunk 1 above)
Comment (driskull) [reply src/components/calcite-action-menu/calcite-action-menu.tsx]: Updated doc

Comment (jcfranco) [reply src/components/calcite-action-menu/calcite-action-menu.tsx]: 🤜🏼 💥 🤛🏼

**Pull request merged**